tox.ini

```diff
@@ -1,12 +1,20 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py27,py34,py35,py36,py37,pypy,flake8
+envlist = py35
```

Comment:
If we're intending this for community use, probably want to keep the higher Python versions and maybe pypy3.